db-commons/src/main/scala/org/bitcoins/db/AppConfig.scala

```diff
@@ -240,6 +235,11 @@ abstract class AppConfig {
     baseDatadir.resolve(lastDirname)
   }
 
+  val slickProfile: String = {
+    ""
+
+  }
```

Comment:
nit: `val slickProfile: String = ""`, also maybe a doc

Reply:
I don't think `slickProfile` is used anywhere, so this can probably be deleted